project/mission-vision.md

```diff
@@ -0,0 +1,14 @@
+## Mission
+
+To provide institutions a foundation to support the scientific data
+life-cycle.
```

Comment:
lifecycle

Reply:
According to dictionaries I've checked on line, "life cycle" should be written as two words without a hyphen. (https://www.dictionary.com/browse/life-cycle)

Reply:
Done.